assets/admin/package.json

```diff
@@ -2,12 +2,17 @@
     "license": "MIT",
     "repository": "https://github.com/sulu/sulu-minimal.git",
     "scripts": {
-        "clean": "rimraf web/admin/build",
+        "clean": "rimraf ./../../web/admin/build",
         "build": "npm run clean && webpack --mode production",
-        "watch": "npm run clean && webpack --mode development -w"
+        "watch": "npm run clean && webpack --mode development -w",
+        "preinstall": "npm install symlink-dir --save-dev && symlink-dir ../../vendor vendor",
+        "postinstall": "symlink-dir node_modules ../../vendor/node_modules"
```

Comment:
as sulu-* is a symlink and when searching for the parent it will never get to the assets/admin folder we symlink the `node_modules` folder of this directory to ../../vendor/node_modules now it will find them correctly.